lib/standard/kernel.nit

```diff
@@ -470,6 +470,36 @@ interface Numeric
 	#     assert (1.9).to_b == 1.to_b
 	fun to_b: Byte is abstract
 
+	# The Int8 equivalent of `self`
+	#
+	#     assert (-1).to_i8 == 0xFFFFi8
+	#     assert (1.9).to_i8 == 1i8
+	fun to_i8: Int8 is abstract
```

Comment:
You can provide a default implementation that does `return self.to_i.to_i8`, same things with other. Ie you use Int as a intermediate representation of integers